### PR TITLE
Suppress npm warnings in production

### LINKED ISSUE #99: Suppress NPM install warnings in production
Since these projects are meant to be temporary and are owned by individual organizations, the security risks are quite low.

Let's suppress the NPM install warnings. I believe we can use `--loglevel=error` for this.

### PR DESCRIPTION
**Pull request by tylers-username**
Adds `--silent` to production installs of the frontend to quiet warnings.

### CODE REVIEW

File: .upsun/config.yaml [hunk 1]
```diff
@@ -23,7 +23,7 @@ applications:
         hooks:
             build: |
                 set -eux
-                npm install
+                npm install --silent
```
Comment (chadwcarlson) [inline .upsun/config.yaml:26]: The `npm_config_loglevel` should already do this @tylers-username

Comment (tylers-username) [reply .upsun/config.yaml:26]: Ah, I didn't realize that had already been included. 

I attempted `npm install --log-level=error` but continued to see the vulnerability warnings

Comment (tylers-username) [reply .upsun/config.yaml:26]: If the build hook no longer shows warnings we can close this. Sorry for the noise 😄

Comment (tylers-username) [reply .upsun/config.yaml:26]: `main` seems alright. 

image


I'll close.

**Pull request closed without merging**